docs/secure-connections/sauce-connect/faq.md

```diff
@@ -6,10 +6,17 @@ sidebar_label: FAQ
 
 If you're new to Sauce Connect Proxy or troubleshooting, this list of frequently asked questions can help you with what you need to know.
 
+
 ## What outbound ports do I need open for Sauce Connect Proxy?
 
 Sauce Connect can only communicate with Sauce Labs via `port 443`. This is not configurable.
 
+## What is the difference between port:443 and --proxy <host:port>?
```

Comment:
Would change the title to when do I need to use port 443 and how does it relate to the `--proxy` flag?